udata/frontend/markdown.py

```diff
@@ -1,18 +1,52 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from urlparse import urlparse
+
 import bleach
 import CommonMark
 from html5lib.sanitizer import HTMLSanitizer
 from flask import current_app, Markup
 from werkzeug.local import LocalProxy
 from jinja2.filters import do_truncate, do_striptags
 
+from udata.i18n import _
+
 md = LocalProxy(lambda: current_app.extensions['markdown'])
 
 EXCERPT_TOKEN = '<!--- --- -->'
 
 
+def avoid_mailto_callback(attrs, new=False):
+    """Remove completely the link containing a `mailto`."""
+    if attrs['href'].startswith('mailto:'):
+        return None
```

Comment:
Why not reuse https://github.com/etalab/udata/blob/master/udata/frontend/helpers.py#L116-L118